### PR TITLE
Create community-systems.md

### PR DESCRIPTION
**Pull request by justinegeffen**
Added new page, migration from docs.

### CODE REVIEW

File: SUMMARY.md [hunk 1]
```diff
@@ -174,6 +174,7 @@
 
 * [Contributors](contributors/contributors/README.md)
   * [Mattermost Community](contributors/contributors/community.md)
+  * [Bugs, Features, and Troubleshooting](contributors/contributors/community/community-systems.md)
```
Comment (jasonblais) [inline SUMMARY.md]:
```suggestion
  * [Community Systems](contributors/contributors/community/community-systems.md)
```

File: contributors/contributors/community-systems.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
+# Bugs, Feature Ideas, Troubleshooting
```
Comment (jasonblais) [inline contributors/contributors/community-systems.md]:
```suggestion
# Community Systems
```

**Review by jasonblais — DISMISSED** [review]

**Comment by jasonblais** [comment]
@justinegeffen I made major edits, removed most of the sections related to features and improvements as they're covered by our new [product planning docs](https://handbook.mattermost.com/operations/research-and-development/product/product-planning), and removed the GitHub workflow section as it was largely outdated and covered by https://developers.mattermost.com/

**Comment by justinegeffen** [comment]
Thanks Jason, that works for me. :)

**Review by jasonblais — APPROVED** [review]

**Pull request merged**